### PR TITLE
Process map elements in knowledge panel

### PR DESCRIPTION
**Pull request by jasmeet0817**

### CODE REVIEW

**Review by monsieurtanuki — COMMENTED** [review]
Hi @jasmeet0817!
Of course `Map` can have different meanings (hashmap or world map), but I imagine there are more relevant names than `KnowledgePanelMapElement`. Neither `Element` sounds nice: I assumed it was a hashmap item.
What about `KnowledgePanelGeoLocations`?
(my 2 cents...)

**Comment by jasmeet0817** [comment]
> Hi @jasmeet0817! Of course `Map` can have different meanings (hashmap or world map), but I imagine there are more relevant names than `KnowledgePanelMapElement`. Neither `Element` sounds nice: I assumed it was a hashmap item. What about `KnowledgePanelGeoLocations`? (my 2 cents...)

That is indeed a good point, but the thing is the element type is `map` so we should keep `Map` somewhere in the name, and also `Element`. `Element` is present as the suffix of all such objects that are specific types of KnowledgePanel elements.

So I changed it to `WorldMap`, also made some other changes, hope they make you happy :)

File: lib/model/KnowledgePanelElement.dart [hunk 1]
```diff
@@ -180,6 +180,57 @@ class KnowledgePanelTableColumn extends JsonObject {
...
+      _$KnowledgePanelWorldMapElementFromJson(json);
+
+  @override
+  Map<String, dynamic> toJson() => _$KnowledgePanelWorldMapElementToJson(this);
+}
+
+/// Element representing a geo location of a map pointer.
+@JsonSerializable()
+class KnowledgePanelGeoPointer extends JsonObject {
+  final KnowledgePanelLatLng? geo;
```
Comment (monsieurtanuki) [inline lib/model/KnowledgePanelElement.dart:202]: Can it be `null`? What would be the point, then?

**Review by monsieurtanuki — APPROVED** [review]
@jasmeet0817 Feel free to ignore my comments.

File: lib/model/KnowledgePanelElement.dart (hunk 1 above)
Comment (jasmeet0817) [reply lib/model/KnowledgePanelElement.dart:202]: yeah Stephane said it could have other information (non-geo). I actually don't know exactly what yet

**Pull request merged**